api/nakadi-event-bus-api.yaml

```diff
@@ -827,135 +685,118 @@ definitions:
-              perspective of the producer. Consumers MIGHT use this value to assert uniqueness of
-              reception of the Event.
-            type: string
-            format: uuid
-            example: '105a76d8-db49-4144-ace7-e683e8f4ba46'
-          event_type:
-            description: |
-              The EventType of this Event. This is enriched by Nakadi on reception of the Event
-              based on the endpoint where the Producer sent the Event to.
-
-              If provided MUST match the endpoint. Failure to do so will cause rejection of the
-              Event.
-            type: string
-            example: 'pennybags.payment-business-event'
-          occurred_at:
-            description: |
-              Timestamp of creation of the Event generated by the producer.
-            type: string
-            format: date-time
-            example: '1996-12-19T16:39:57-08:00'
-          received_at:
-            type: string
-            description: |
-              Timestamp of the reception of the Event by Nakadi. This is enriched upon reception of
-              the Event.
-              If set by the producer Event will be rejected.
-            format: date-time
-            example: '1996-12-19T16:39:57-08:00'
-          parent_eids:
-            type: array
-            items:
-              type: string
-              format: uuid
-              description: |
-                Event identifier of the Event that caused the generation of this Event.
-                Set by the producer.
-              example: '105a76d8-db49-4144-ace7-e683e8f4ba46'
-          flow_id:
-            description: |
-              The flow-id of the producer of this Event. As this is usually a HTTP header, this is
-              enriched from the header into the metadata by Nakadi to avoid clients having to
-              explicitly copy this.
-            type: string
-            example: 'JAh6xH4OQhCJ9PutIV_RYw'
-          partition:
-            description: |
-              Indicates the partition assigned to this Event.
+      eid:
+        description: |
+          Identifier of this Event.
 
-              Required to be set by the client if partition strategy of the EventType is
-              'user_defined'.
-            type: string
-            example: '0'
-        required:
-          - eid
-          - occurred_at
+          Clients MUST generate this value and it SHOULD be guaranteed to be unique from the
+          perspective of the producer. Consumers MIGHT use this value to assert uniqueness of
+          reception of the Event.
+        type: string
+        format: uuid
+        example: '105a76d8-db49-4144-ace7-e683e8f4ba46'
+      event_type:
+        description: |
+          The EventType of this Event. This is enriched by Nakadi on reception of the Event
+          based on the endpoint where the Producer sent the Event to.
 
-        additionalProperties: false
```

Comment:
This `additionalProperties` can go; it breaks swagger tooling and there's little benefit to closing metadata for extension (we can just use must ignore processing)